_posts/2019-11-27-homebrew-2.2.0.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 2.2.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.2.0. The most significant changes since 2.1.0 are macOS Catalina support, performance increases and better Homebrew on Linux ecosystem integration.
+
+Major changes and deprecations since 2.1.0:
+
+- [macOS Catalina (10.15) is supported. macOS Sierra (10.12) and older are unsupported.](https://github.com/Homebrew/brew/pull/6500)
+- [The no-op case for HOMEBREW_AUTO_UPDATE_SECS is dramatically faster and defaults to 5 minutes (rather than 1).](https://github.com/Homebrew/brew/pull/6532)
+- [`brew upgrade` no longer has an unsuccessful error code if the formula is up-to-date.](https://github.com/Homebrew/brew/pull/6728)
+- [`brew upgrade`'s post-install dependent checking is dramatically faster and more reliable.](https://github.com/Homebrew/brew/pull/6698)
+- [Homebrew on Linux raised their minimum requirements.](https://github.com/Homebrew/brew/pull/6137)
+- <https://formulae.brew.sh> [displays Linux formulae](https://github.com/Homebrew/formulae.brew.sh/pull/150). [`brew info` outputs Linux analytics based on this data when on Linux.](https://github.com/Homebrew/brew/pull/6639)
```

Comment:
```suggestion
- <https://formulae.brew.sh> [displays Linux formulae](https://github.com/Homebrew/formulae.brew.sh/pull/150). When on Linux, [`brew info` outputs Linux analytics data](https://github.com/Homebrew/brew/pull/6639).
```

Rather than having a whole sentence be a link?